ultravox/data/datasets.py

```diff
@@ -575,6 +605,99 @@ def _get_sample(self, row: transformers.BatchFeature) -> VoiceSample:
         )
 
 
+class HeySQuADHumanDataset(QAVoiceDatasetMixin):
+    """
+    HeySQuAD is a large-scale Spoken Question Answering (SQA) dataset which includes 76k human-spoken questions,
+    97k machine-generated questions, and their corresponding textual answers from the SQuAD QA dataset.
+    https://arxiv.org/abs/2304.13689
+
+    This dataset is the human-spoken version of HeySQuAD.
+    """
+
+    def __init__(self, args: VoiceDatasetArgs) -> None:
+        super().__init__(args)
+        dataset = self._load_audio_dataset(
+            "yijingwu/HeySQuAD_human", split=args.split.value
+        )
+        self._init_dataset(dataset)
+
+    def _get_sample(self, row: transformers.BatchFeature) -> Optional[VoiceSample]:
+        """
+        Example conversation
+            <|user|> Context: {context}
+            Question: {question}
+            <|assistant|> {answer}
+        """
+        if row["is_impossible"] or not row["answers"]:
+            # Skip samples with no answer
+            return None
+        if len(row["context"]) > self._args.max_context_length:
+            # Skip samples with long context
+            return None
+
+        prompt = self._get_query_prompt(
+            question_str=row["question"], context=row["context"]
+        )
+        messages = [
+            {"role": "user", "content": prompt},
+            {"role": "assistant", "content": row["answers"][0]["text"]},
+        ]
+        return self._make_sample(
+            messages, self._get_audio(row), audio_transcript=row["question"]
+        )
+
+
+class SlueSQA5Dataset(QAVoiceDatasetMixin):
+    """
+    SLUE-SQA-5 Dataset contains question texts, question audio, answer text, document text, and document audio from these datasets:
+      * SQuAD1.1 (for questions whose question_id starts with 'squad-')
+      * Natural Questions (for questions whose question_id starts with 'nq-')
+      * TriviaQA (for questions whose question_id starts with 'triviaqa-')
+    The following datasets are supposed to be included, but I haven't found them everywhere:
+      * WebQuestions (for questions whose question_id starts with 'wq-')
+      * CuratedTREC (for questions whose question_id starts with 'trec-')
+      * Spoken Wikipedia
+
+
+    Splits: train, validation, test, verified_test
+    """
+
+    BASE_AUDIO_COLUMNS = ["question_audio", "document_audio"]
```

Comment:
is this used?

Reply:
Do you mean is `BASE_AUDIO_COLUMNS` ever used or are you asking whether we need to convert `document_audio` if we never use it?

![image](https://github.com/user-attachments/assets/7780ebe2-adb2-49b9-bb98-d8f30be4b9d0)

Reply:
I was being a bit wasteful by keeping `document_audio` at the cost of making sure we don't forget (e.g. if a child-class wants to use it). Didn't have a strong opinion here though.

Reply:
OK, I didn't realize this was overriding a base class constant. A little confusing perhaps but we can worry about this later.